Training25/Program.cs

```diff
@@ -3,11 +3,29 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T4 branch.
+// Program to print chess board.
 // ------------------------------------------------------------------------------------------------
+using System.Text;
+using static System.Console;
 namespace Training25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      OutputEncoding = new UnicodeEncoding ();
+      for (int i = 0; i < 17; i++)
+         switch (i) {
+            case 0: WriteLine ("┏━━━┳━━━┳━━━┳━━━┳━━━┳━━━┳━━━┳━━━┓"); break;
+            case 1: PrintChars (['♜', '♞', '♝', '♛', '♚', '♝', '♞', '♜']); break;
+            case 3: PrintChars ([.. Enumerable.Repeat ('♟', 8)]); break;
+            case 5 or 7 or 9 or 11: PrintChars ([.. Enumerable.Repeat (' ', 8)]); break;
+            case 13: PrintChars ([.. Enumerable.Repeat ('♙', 8)]); break;
+            case 15: PrintChars (['♖', '♘', '♗', '♕', '♔', '♗', '♘', '♖']); break;
+            case 16: WriteLine ("\n┗━━━┻━━━┻━━━┻━━━┻━━━┻━━━┻━━━┻━━━┛"); break;
+            default: WriteLine ("\n┣━━━╋━━━╋━━━╋━━━╋━━━╋━━━╋━━━╋━━━┫"); break;
+         }

```

Comment:
- The top and bottom lines of the board appear first and last and therefore need not be inside the switch case.
- Instead of a char array, you can simply have a string and do a `.ToCharArray ()` in `string.Join`
```suggestion
         PrintChars (['♜', '♞', '♝', '♛', '♚', '♝', '♞', '♜']);
         PrintRow ("♜♞♝♛♚♝♞♜");
```
- Repetitive code using `Enumerable.Repeat` for pawns and spaces
- Try and see if you can implement this without using `switch case` with just `for` loop